HISTORY.md

```diff
@@ -4,18 +4,23 @@
 This incorporates all changes after 1.3.5 up to 1.3.6.
 
   * Add support for returned, rejected Promises to `router.param`
+  * **BREAKING**: deps: `path-to-regexp@6.2.0` (See [release changelog](https://github.com/pillarjs/path-to-regexp/releases/tag/v6.2.0))
+    - Support for **named capturing groups** in paths using `RegExp`, i.e.: `/\/(?<group>.+)/` would result in `req.params.group` being populated.
+    - Custom **prefix and suffix groups** using `{}`, e.g.: `/:entity{-:action}?` would match `/user` and `/user-delete`.
+    - Unbalanced patterns now produce an error: `/test(foo` previously worked, but now it expects `(` to be escaped for the previous behavior, e.g. `/test\\(foo`.
```

Comment:
The release notes referenced above says the previous behavior can be achieved with `/test\(foo` but here it states `/test\\(foo`. Is there a reason for the difference? Apologies if I missed something obvious!

Reply:
Hey @xjamundx 👋🏼 Thanks for chiming in.

You're absolute right, good catch. Yeah, I think `path-to-regexp` release notes are wrong, or misleading at the very least. `/test(foo` no longer works because it starts a capturing group that it's never closed - but if you want the literal `(` to match you'd need to escape it with '\'. This being JS, escaping non-special chars with `\` just echoes the char bar (i.e.: `'\('` -> `'('`). So in order to forward the escaped sequence to the underlying `RegExp`, you'd need to escape the backslash as well.

See it in action:

```js
> const { pathToRegexp } = require('path-to-regexp')

> pathToRegexp('/test\(foo').test('/test(foo')
Uncaught TypeError: Unbalanced pattern at 5

> pathToRegexp('/test\\(foo').test('/test(foo')
true
```

In short, I think `router`'s description makes more sense and should cause less confusion.